### PR TITLE
Merge tests and load Json from files

### PR DESCRIPTION
**Pull request by lucidd**
This merges the tests for the `SaltStackParser` class together into on class.
In addition to that i put the json in files with proper formatting so it is easier edit and verify those strings.

### CODE REVIEW

File: src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @Test
+    public void testSaltStackTokenParser() throws Exception {
+        InputStream is = this.getClass().getResourceAsStream("/login_response.json");
+        SaltStackResult<List<SaltStackToken>> result = SaltStackParser.TOKEN.parse(is);
+        assertNotNull(result);
+        assertEquals("", "salt", result.getResult().get(0).getUser());
+        assertEquals("", "pam", result.getResult().get(0).getEauth());
+        assertEquals("", "f248284b655724ca8a86bcab4b8df608ebf5b08b", result.getResult().get(0).getToken());
+    }
+}
```
Comment (moio) [inline src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java]: Please add newlines at the end of files :-)

Comment (lucidd) [inline src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java]: Done :)

File: src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+    @Test
+    public void testSaltStackStringParser() throws Exception {
+        InputStream is = this.getClass().getResourceAsStream("/logout_response.json");
+        SaltStackResult<String> result = SaltStackParser.STRING.parse(is);
+        assertNotNull(result);
+    }
+
+
```
Comment (renner) [inline src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java]: Please put only one newline between those methods and at the beginning of the class. Thanks!

Comment (lucidd) [inline src/test/java/com/suse/saltstack/netapi/parser/SaltStackParserTest.java]: Done :) also removed the unused import.

**Comment by moio** [comment]
Very good, thanks again for this contribution.

**Pull request merged**